packages/taquito-signer/test/taquito-signer.spec.ts

```diff
@@ -183,4 +184,49 @@ describe('inmemory-signer', () => {
     );
     done();
   });
+
+  it('Intantiate tz1 from mnemonic from in memory signer', async (done) => {
```

Comment:
typo 'instantiate'

also for consistency, let's still use the 'should..' form as the unit test name prefix